### ISSUE TITLE
[MLA] minio installation not working with Helm 3.13.1

### ISSUE THREAD

**Issue by wurbanski** [kubermatic/kubermatic#12804]
### What happened?
deploying mla minio fails with Helm 3.13.1

```
INFO[0025]    📦 Deploying Minio…
DEBU[0025]       Ensuring namespace…                     namespace=mla
DEBU[0025]       Checking for release…                   name=minio namespace=mla
DEBU[0025] $ KUBECONFIG=./secrets/kubeone-seed-fra-kubeconfig helm --namespace mla list --all -o json
DEBU[0025]       Installing…
DEBU[0025] $ KUBECONFIG=./secrets/kubeone-seed-fra-kubeconfig helm --namespace default repo add dep-minio-0 https://charts.min.io/
DEBU[0026] $ KUBECONFIG=./secrets/kubeone-seed-fra-kubeconfig helm --namespace default dependency build releases/v2.23.5/charts/mla/minio
DEBU[0027] $ KUBECONFIG=./secrets/kubeone-seed-fra-kubeconfig helm --namespace default repo remove dep-minio-0
DEBU[0027] $ KUBECONFIG=./secrets/kubeone-seed-fra-kubeconfig helm --namespace mla upgrade --install --timeout 15m0s --values /tmp/helmvalues.2325240576 --atomic minio releases/v2.23.5/charts/mla/minio
ERRO[0028] ❌ Operation failed: failed to deploy Minio: failed to deploy Helm release: failed to install: WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: ./secrets/kubeone-seed-fra-kubeconfig
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: ./secrets/kubeone-seed-fra-kubeconfig
coalesce.go:286: warning: cannot overwrite table with non table for minio.minio.makeBucketJob.annotations (map[])
coalesce.go:286: warning: cannot overwrite table with non table for minio.minio.makeBucketJob.podAnnotations (map[])
Error: Unable to continue with install: could not get information about the resource ServiceAccount "" in namespace "mla": resource name may not be empty.
```

### Expected behavior
Installation works properly with new Helm releases.

### How to reproduce the issue?

Run installation of mla using Helm 3.13.1

### How is your environment configured?

- KKP version: v2.23
- Shared or separate master/seed clusters?: N/A

### Provide your KKP manifest here (if applicable)

N/A

### What cloud provider are you running on?

N/A

### What operating system are you running in your user cluster?

N/A

### Additional information

Possible fix is to include service account name in values file:

```yaml
minio:
  serviceAccount:
    name: "minio-sa"
```

**Comment by embik** [comment]
This was already fixed on `main` via #12697. We did not consider that for backport at the time. Working on it.

**Comment by embik** [comment]
Fixed via #12806 and #12812. Will be in the next KKP 2.23 and 2.22 patch releases.
